centreon/www/install/php/Update-23.10.2.php

```diff
@@ -110,6 +110,23 @@
     );
 };
 
+$populateDashboardTables = function(CentreonDb $pearDB): void {
```

Comment:
This should be move in 23.10.3 script, as the 23.10.2 is currently being released

Reply:
:')